style/values/specified/font.rs

```diff
@@ -2007,6 +2007,8 @@ impl XTextScale {
     ToCss,
     ToResolvedValue,
     ToShmem,
+    Serialize,
+    Deserialize,
```

Comment:
Nit: not a big deal because this change won't get into Firefox, but I think it's better to preserve compatibility and use this instead:

```rs
#[derive(
    Clone,
    Debug,
    MallocSizeOf,
    PartialEq,
    SpecifiedValueInfo,
    ToComputedValue,
    ToCss,
    ToResolvedValue,
    ToShmem,
)]
#[cfg_attr(feature = "servo", derive(Deserialize, Serialize))]
/// Internal property that reflects the lang attribute
pub struct XLang(#[css(skip)] pub Atom);
```

This will add `Deserialize, Serialize` just for Servo, not for Gecko.

Reply:
It's fine to make stuff serializable in Gecko generally, fwiw.

Reply:
(Unless Atom is somehow not serializable in Firefox for some reason)

Reply:
Yeah Atom not serializable in Firefox:
```
 0:16.17 error[E0277]: the trait bound `Atom: Serialize` is not satisfied
 0:16.18     --> servo/components/style/values/specified/font.rs:2006:5
 0:16.18      |
 0:16.18 2006 |     Serialize,
 0:16.18      |     ^^^^^^^^^ the trait `Serialize` is not implemented for `Atom`
 0:16.18 ...
 0:16.18 2010 | pub struct XLang(#[css(skip)] pub Atom);
 0:16.18      |                  --------------------- required by a bound introduced by this call
```